guides/common/modules/ref_supported-clients-in-registration.adoc

```diff
@@ -0,0 +1,79 @@
+[id="supported-clients-in-registration_{context}"]
+= Supported Clients in Registration
+
+{Project} supports the following operating systems and architectures for registration.
+
+ifndef::orcharhino,satellite[]
+[NOTE]
+====
+The following combinations have been tested.
+https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/registration/global_registration.erb[*Global registration*] is a provisioning template for a shell script, which can be extended for additional systems.
+
+If you decide to extend the template, please submit your changes to https://github.com/theforeman/foreman/#how-to-contribute[our repository].
+Thanks for your contribution!
+====
+endif::[]
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifndef::satellite[]
+* AlmaLinux
+* Amazon Linux
+* CentOS
+endif::[]
+ifndef::satellite[]
+* Debian
+ifndef::orcharhino[]
+* Fedora
+* FreeBSD
+* OpenSUSE
+endif::[]
+* Oracle Linux
+endif::[]
+* {EL} 9, 8, 7
+ifdef::satellite[]
+* {EL} 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+endif::[]
+ifndef::satellite[]
+* {SLES}
+ifndef::orcharhino[]
+* Solaris
```

Comment:
I doubt someone tested this and given it's broken on FreeBSD I'd also assume it's broken on Solaris. In the past people have used Foreman & Solaris by using Puppet, but I'm not aware of anyone using it with registration.
```suggestion
```